components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -152,17 +165,19 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
     return (i + 1) < conditions.length;
   }
 
-  showDelete(i: number): boolean {
+  showDelete(): boolean {
     const conditions = this.ruleForm.get('conditions') as FormArray;
-    return (i + 1) < conditions.length;
+    return conditions.length > 1;
   }
 
   populateConditions() {
     this.conditions = [];
 
     if (this.rule.conditions && this.rule.conditions.length !== 0) {
       this.rule.conditions.forEach(c => {
-        this.conditions.push(this.createCondition(c.attribute, c.operator, c.values));
+        this.conditions.push(
+          // Convert values array to display string
+          this.createCondition(c.attribute, c.operator, c.values.join(', ')));
```

Comment:
Remember my story about conditions earlier? Fairly straightforward now. Here's one direction, taking the storage object and converting it to a string for display.